### ISSUE TITLE
captcha not showing along with comment form

### ISSUE THREAD

**Issue by michaelh99** [pstimpel/phpcaptchapiwigo#2]
There's no specific checkbox to enable it and it's not appearing even when otherwise enabled

https://www.dragonseye.com/photos/picture.php?/23716-023__mg_4115/categories

**Comment by michaelh99** [comment]
Also, when I try to leave a comment I see the following errors at the top of the page


Notice: Undefined index: captcha_hash in /home/michael9/public_html/photos/plugins/phpcaptchapiwigo/include/picture.inc.php on line 27

Notice: Undefined index: captcha_code in /home/michael9/public_html/photos/plugins/phpcaptchapiwigo/include/picture.inc.php on line 30

Notice: Undefined index: captcha_code in /home/michael9/public_html/photos/plugins/phpcaptchapiwigo/include/picture.inc.php on line 34

**Comment by CharlesButcher** [comment]
@michaelh99, I think it's an issue with your Piwigo theme, which If I'm not mistaken is "Bootstrap Default". I used the same theme and I had exactly the same issue. Same PHP notices, plus two more for `Cannot modify header information`. Changing the theme allowed the captcha to appear. I tested only a couple of themes: "clear" and "Pure_grey", and both worked.

The Bootstrap theme looks good, so it would be nice to get it working with captchas (I tried a couple of other captcha plugins and they failed too). But that's beyond my coding skills. The developer of Bootstrap Default has [acknowledged a bug](https://github.com/Philio/bootstrapdefault/issues/47).
